driver-core/src/main/java/com/datastax/driver/core/policies/Clock.java

```diff
@@ -0,0 +1,16 @@
+package com.datastax.driver.core.policies;
+
+/**
+ * Wraps System.nanoTime() to make it easy to mock in tests.
+ */
+class Clock {
```

Comment:
This new interface somewhat overlaps with `com.datastax.driver.core.Clock` and thus makes the improvements brought by JAVA-727 unusable within this new policy.